x-pack/plugins/apm/public/components/app/Settings/anomaly_detection/jobs_list.tsx

```diff
@@ -0,0 +1,156 @@
+
+const columns: Array<ITableColumn<AnomalyDetectionJobByEnv>> = [
+  {
+    field: 'service.environment',
+    name: i18n.translate(
+      'xpack.apm.settings.anomalyDetection.jobList.environmentColumnLabel',
+      { defaultMessage: 'Environment' }
+    ),
+    render: (environment: string) => environment,
+  },
+  {
+    field: 'job_id',
+    align: 'right',
+    name: i18n.translate(
+      'xpack.apm.settings.anomalyDetection.jobList.actionColumnLabel',
+      { defaultMessage: 'Action' }
+    ),
+    render: (jobId: string) => (
+      <MLJobLink jobId={jobId}>
+        {i18n.translate(
+          'xpack.apm.settings.anomalyDetection.jobList.mlJobLinkText',
+          {
+            defaultMessage: 'View job in ML',
+          }
+        )}
+      </MLJobLink>
+    ),
+  },
+];
+
+interface Props {
+  isLoading: boolean;
+  hasFetchFailure: boolean;
+  onAddEnvironments: () => void;
+  anomalyDetectionJobsByEnv: AnomalyDetectionJobByEnv[];
+}
+export const JobsList = ({
+  isLoading,
+  hasFetchFailure,
+  onAddEnvironments,
+  anomalyDetectionJobsByEnv,
+}: Props) => {
+  return (
+    <EuiPanel>
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <EuiTitle>
+            <h2>
+              {i18n.translate(
+                'xpack.apm.settings.anomalyDetection.jobList.environments',
+                {
+                  defaultMessage: 'Environments',
+                }
+              )}
+            </h2>
+          </EuiTitle>
+        </EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiButton fill onClick={onAddEnvironments}>
+            {i18n.translate(
+              'xpack.apm.settings.anomalyDetection.jobList.addEnvironments',
+              {
+                defaultMessage: 'Add environments',
+              }
+            )}
+          </EuiButton>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <EuiSpacer size="l" />
+      <EuiText>
+        <FormattedMessage
+          id="xpack.apm.settings.anomalyDetection.jobList.mlDescriptionText"
+          defaultMessage="You can {mlJobsLink}."
+          values={{
+            mlJobsLink: (
+              <MLLink path="jobs">
+                {i18n.translate(
+                  'xpack.apm.settings.anomalyDetection.jobList.mlDescriptionText.mlJobsLinkText',
+                  {
+                    defaultMessage:
+                      'manage the anomaly detection jobs in Machine Learning',
+                  }
+                )}
+              </MLLink>
+            ),
+          }}
+        />
+      </EuiText>
+      <EuiSpacer size="l" />
+      <ManagedTable
+        noItemsMessage={
+          isLoading ? (
+            <LoadingStatePrompt />
+          ) : hasFetchFailure ? (
+            <FailureStatePrompt />
+          ) : (
+            <EmptyStatePrompt />
+          )
+        }
+        columns={columns}
+        items={isLoading || hasFetchFailure ? [] : anomalyDetectionJobsByEnv}
+        pagination={false}
```

Comment:
I'm not 100% sure what `paginate=false` does. But I assume it removes the pagination controls. Aren't they necessary if many results are returned?

Btw. I agree that we don't have to add complex logic for paginating server side - just talking about client side since it looks like it just works out of the box.